transitracker/tests/database_tests.py

```diff
@@ -0,0 +1,47 @@
+from flask.ext.testing import TestCase
+from transitracker import db
+
+
+class DatabaseTest(TestCase):
+
+    SQLALCHEMY_DATABASE_URI = "sqlite://transitracker_testing"
+    TESTING = True
+
```

Comment:
There are no comments in this file. I'm not sure what these tests are supposed to return, and how we know if the tests are successful. When is this run? We need testing instructions as well as the tests.